components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ImportUtils.java

```diff
@@ -593,10 +602,10 @@ public static JsonObject setCurrentProviderToContext(JsonObject jsonObject, Stri
      * @param pathToArchive Path to API or API Product archive
      * @throws IOException If an error occurs while reading the file
      */
-    public static String getAPIDefinitionAsJson(String pathToArchive) throws IOException {
+    public static String getFileContentAsJson(String pathToArchive) throws IOException {
         String jsonContent = null;
-        String pathToYamlFile = pathToArchive + ImportExportConstants.YAML_API_FILE_LOCATION;
-        String pathToJsonFile = pathToArchive + ImportExportConstants.JSON_API_FILE_LOCATION;
+        String pathToYamlFile = pathToArchive + ImportExportConstants.YAML_EXTENSION;
+        String pathToJsonFile = pathToArchive + ImportExportConstants.JSON_EXTENSION;
```

Comment:
I think we can remove the constants "YAML_API_FILE_LOCATION" and "JSON_API_FILE_LOCATION" from the **ImportExportConstants.java** file. Shall we check whether there are any other usages and remove these if not used in any other place?

Reply:
Fixed with 2adb0a8ed29a163835d83b3c7f1d3f809b27c2d7